examples/tornjak/keycloak/README.md

```diff
@@ -0,0 +1,74 @@
+# Deploy Tornjak with Auth Enabled
+
+To install Spire with the least privileges possible we deploy spire across 2 namespaces.
+
+- Install SPIRE CRDs and deploy SPIRE with Tornjak Enabled
+```shell
+# install SPIRE CRDs
+helm upgrade --install --create-namespace -n spire-mgmt spire-crds charts/spire-crds
+
+
+# deploy SPIRE with Tornjak enabled and provide auth url to enable auth
+helm upgrade --install \
+--set global.spire.namespaces.create=true \
+--values examples/production/values.yaml \
+--values examples/production/example-your-values.yaml \
+--values examples/tornjak/values.yaml \
+--values examples/tornjak/values-auth.yaml \
+--render-subchart-notes spire charts/spire
+
+# test the Tornjak deployment
+helm test spire
+```
+
+- Deploy Keycloak
+```shell
+# create a secret from the realm json file for tornjak realm import
+kubectl create secret generic realm-secret -n spire-server --from-file=examples/tornjak/keycloak/tornjak-realm.json
+
+
+# deploy keycloak as an auth service
+helm upgrade --install --create-namespace -n spire-server keycloak --values examples/tornjak/keycloak/values.yaml oci://registry-1.docker.io/bitnamicharts/keycloak --render-subchart-notes
+```
+
+## Access Tornjak
+
+To access Tornjak use port-forwarding or check the ingress option below.
+
+Run following commands from your shell, if you run with different values your namespace might differ. Consult the install notes printed when running above `helm upgrade` command in that case.
+
+Since `port-forward` is a blocking command, execute them in three different consoles (One for backend, one for frontend and one for auth):
```

Comment:
```suggestion
Since `port-forward` is a blocking command, execute them in three different consoles (one for backend, one for frontend, and one for auth):
```